app/views/idv/inherited_proofing_cancellations/destroy.html.erb

```diff
@@ -0,0 +1,8 @@
+<% title t('titles.inherited_proofing.cancelled') %>
+
+<%= render StatusPageComponent.new(status: :error) do |c| %>
+  <% c.header { t('inherited_proofing.cancel.headings.confirmation.hybrid') } %>
+
+  <p><%= t('inherited_proofing.cancel.instructions.switch_back') %></p>
+  <%= image_tag(asset_url('inherited_proofing/switch.png'), width: 193, alt: t('inherited_proofing.cancel.instructions.switch_back_image')) %>
```

Comment:
Do we need a second copy of this image, or could we have used the existing `idv/switch.png` ?

Reply:
Sorry, I didn't see this, looking back on it. Yea, was trying to make a clear distinction because there were no figma's for these yet; hindsight, that's probably more reason _to_ reuse it.